tests/helper/helper_generic.go

```diff
@@ -143,3 +143,25 @@ func WatchNonRetCmdStdOut(cmdStr string, timeout time.Duration, check func(outpu
 		}
 	}
 }
+
+// WaitTillUrlexist wait for the given url to be up on the cluster
+func WaitTillUrlexist(namespace, url string, timeoutMinutes int) bool {
```

Comment:
We don't need this function. We already have a similar generic function https://github.com/openshift/odo/blob/3587b9eac2501ffc29d54207368af591424cb0be/tests/helper/helper_generic.go#L40